iOS-Settings/Sources/App/DetailViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DetailViewController.swift
+//  iOS-Settings
+//
+//  Created by Kuanysh al-Khattab Auyelgazy on 03.03.2023.
+//
+
+import UIKit
+import SnapKit
+
+class DetailViewController: UIViewController {
+
+    var setting: Setting?
+
+    // MARK: - Outlets
+
+    private lazy var imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = setting?.iconImage
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .lightGray
+        title = setting?.name
+        setupHierarchy()
```

Comment:
create func to setup view. view.backgroundColor = .lightGray and title = setting?.name should go to the new func